spec/truffle.mspec

```diff
@@ -148,8 +153,7 @@ class MSpecScript
   set :files, get(:language) + get(:core) + get(:library) + get(:truffle)
 end
 
-is_child_process = ENV.key? "MSPEC_RUNNER"
-if i = ARGV.index('slow') and ARGV[i-1] == '--excl-tag' and is_child_process
+if i = ARGV.index('slow') and ARGV[i-1] == '--excl-tag' and MSpecScript.child_process?
```

Comment:
We don't, please add one if you think it matters.
I don't want to change more existing code than needed in this PR.